provider/awssd/aws_sd_test.go

```diff
@@ -38,6 +39,10 @@ import (
 // Compile time check for interface conformance
 var _ AWSSDClient = &AWSSDClientStub{}
 
+var (
+	ErrNamespaceNotFound = errors.New("Namespace not found")
+)
```

Comment:
```suggestion
const ErrNamespaceNotFound = errors.New("Namespace not found")
```

Reply:
Hey, are you sure about this one?  I thought `error` could not be a `const` 🤔

Reply:
ah you're right. It works with many languages but not in Go.